utils/config.go

```diff
@@ -51,21 +50,24 @@ func FindConfigFile(fileName string) (path string) {
 	return ""
 }
 
+// FindDir recurses to the root looking for the given directory, falling back to `./` if not found.
 func FindDir(dir string) (string, bool) {
-	fileName := "."
-	found := false
-	if _, err := os.Stat("./" + dir + "/"); err == nil {
-		fileName, _ = filepath.Abs("./" + dir + "/")
-		found = true
-	} else if _, err := os.Stat("../" + dir + "/"); err == nil {
-		fileName, _ = filepath.Abs("../" + dir + "/")
-		found = true
-	} else if _, err := os.Stat("../../" + dir + "/"); err == nil {
-		fileName, _ = filepath.Abs("../../" + dir + "/")
-		found = true
+	cwd, _ := os.Getwd()
+
+	for {
+		foundDir := filepath.Join(cwd, dir)
+		if _, err := os.Stat(foundDir); err == nil {
+			return foundDir, true
+		}
+
+		parent := filepath.Join(cwd, "../")
+		if parent == cwd {
+			break
+		}
+		cwd = parent
 	}
 
-	return fileName + "/", found
+	return "./", false
```

Comment:
These semantics are a bit weird, but I've preserved (and documented) them for clients assuming same.